go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/oapi-codegen/runtime v1.1.1
 	github.com/operator-framework/api v0.27.0
 	github.com/operator-framework/operator-lifecycle-manager v0.27.0
-	github.com/percona/everest-operator v0.6.0-dev1.0.20250317191108-0a7d4b5c8d3b
+	github.com/percona/everest-operator v0.6.0-dev1.0.20250318175140-0d5356fffa59
```

Comment:
Don't forget to update this after https://github.com/percona/everest-operator/pull/696 is merged.